### ISSUE TITLE
Update matplotlib

### ISSUE THREAD

**Issue by kkk669** [omz/Pythonista-Issues#547]
Matplotlib 1.4 is bundled with Pythonista 3.2, but some Python modules, such as chainer 4, depend on Matplotlib 2.

Is there a plan to update bundled Matplotlib?

**Comment by cclauss** [comment]
Yes.  Pythonista has tons of modules that could use an update...
* https://pyup.io/account/repos/github/cclauss/pythonista-module-versions

**Comment by kkk669** [comment]
Related to #550
